build.mill.scala

```diff
@@ -281,22 +280,31 @@ object ci extends Module {
   @unused
   def publishSonatype(tasks: mill.main.Tasks[PublishModule.PublishData]): Command[Unit] =
     Task.Command {
+      val publishVersion = publishVersion0()
+      System.err.println(s"Publish version: $publishVersion")
+      val bundleName = s"org.virtuslab.scala-cli.java-class-name-java-class-name:$publishVersion"
```

Comment:
```suggestion
      val bundleName = s"org.virtuslab.scala-cli.java-class-name-java-class-name-$publishVersion"
```
not sure if that matters, but suggested in https://mill-build.org/mill/scalalib/publishing.html#_publishall

Reply:
let's go with your suggestion and see if it works